sal_interfaces/Forecast/Forecast_Telemetry.xml

```diff
@@ -0,0 +1,674 @@
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast gust</Description>
+      <Units>kilometer/hour</Units>
+      <EFDB_Name>gust</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>long</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast low clouds</Description>
+      <Units>pct</Units>
+      <EFDB_Name>low_clouds</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>long</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast mid clouds</Description>
+      <Units>pct</Units>
+      <EFDB_Name>mid_clouds</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>long</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast high clouds</Description>
+      <Units>pct</Units>
+      <EFDB_Name>high_clouds</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>long</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast sunshine time</Description>
+      <Units>minute</Units>
+      <EFDB_Name>sunshine_time</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>long</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast visibility</Description>
+      <Units>meter</Units>
+      <EFDB_Name>visibility</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast skin temperature</Description>
+      <Units>deg</Units>
+      <EFDB_Name>skin_temperature</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast dew point temperature</Description>
+      <Units>deg</Units>
+      <EFDB_Name>dew_point_temperature</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>long</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast precipitation probability</Description>
+      <Units>pct</Units>
+      <EFDB_Name>precipitation_probability</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>long</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast cape</Description>
+      <Units>Joule*kilogram</Units>
+      <EFDB_Name>cape</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast lifted index</Description>
+      <Units>unitless</Units>
+      <EFDB_Name>lifted_index</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast evapotransipiration</Description>
+      <Units>unitless</Units>
+      <EFDB_Name>evapo_transipiration</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast reference evaporated transipiration fao</Description>
+      <Units>unitless</Units>
+      <EFDB_Name>reference_evapo_transipiration_fao</EFDB_Name>
+    </item>
+  </SALTelemetry>
+  <SALTelemetry>
+    <Subsystem>Forecast</Subsystem>
+    <EFDB_Topic>Forecast_trend_day</EFDB_Topic>
```

Comment:
Please rename this to `Forecast_dailyTrend`. Note that all comments, that apply to the items of `Forecast_hourlyTrend`, apply to the items of `Forecast_dailyTrend`.